python_modules/libraries/dagster-components/setup.py

```diff
@@ -34,7 +34,7 @@ def get_version() -> str:
         "Operating System :: OS Independent",
     ],
     packages=find_packages(exclude=["dagster_components_tests*", "examples*"]),
-    install_requires=["dagster>=1.9.5", "typer"],
+    install_requires=["dagster>=1.9.5", "typer", "tomlkit"],
```

Comment:
it looks like tomlkit is only used in testst currently. Do you anticipate that changing/should this be in a tests extra for now?